[2022-06-13 12:35:24,047] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: gcs_2_bq_dag.bq_create_fhvhv_tripdata_partitioned_table_task scheduled__2022-06-12T00:00:00+00:00 [queued]>
[2022-06-13 12:35:24,061] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: gcs_2_bq_dag.bq_create_fhvhv_tripdata_partitioned_table_task scheduled__2022-06-12T00:00:00+00:00 [queued]>
[2022-06-13 12:35:24,063] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 12:35:24,067] {taskinstance.py:1358} INFO - Starting attempt 7 of 7
[2022-06-13 12:35:24,068] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 12:35:24,088] {taskinstance.py:1378} INFO - Executing <Task(BigQueryInsertJobOperator): bq_create_fhvhv_tripdata_partitioned_table_task> on 2022-06-12 00:00:00+00:00
[2022-06-13 12:35:24,101] {standard_task_runner.py:52} INFO - Started process 14666 to run task
[2022-06-13 12:35:24,110] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'gcs_2_bq_dag', 'bq_create_fhvhv_tripdata_partitioned_table_task', 'scheduled__2022-06-12T00:00:00+00:00', '--job-id', '655', '--raw', '--subdir', 'DAGS_FOLDER/gcs_to_bq_dag.py', '--cfg-path', '/tmp/tmpnjdlpddd', '--error-file', '/tmp/tmp4rg0sqsh']
[2022-06-13 12:35:24,113] {standard_task_runner.py:80} INFO - Job 655: Subtask bq_create_fhvhv_tripdata_partitioned_table_task
[2022-06-13 12:35:24,192] {task_command.py:370} INFO - Running <TaskInstance: gcs_2_bq_dag.bq_create_fhvhv_tripdata_partitioned_table_task scheduled__2022-06-12T00:00:00+00:00 [running]> on host 0825143a1170
[2022-06-13 12:35:24,375] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=bq_create_fhvhv_tripdata_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=7
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T00:00:00+00:00
[2022-06-13 12:35:24,383] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-06-13 12:35:24,385] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-06-13 12:35:24,390] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-06-13 12:35:24,392] {bigquery.py:2148} INFO - Executing: {'query': {'query': 'CREATE_BQ_TBL_QUERY_fhvhv', 'useLegacySql': False}}
[2022-06-13 12:35:24,407] {bigquery.py:1538} INFO - Inserting job ***_gcs_2_bq_dag_bq_create_fhvhv_tripdata_partitioned_table_task_2022_06_12T00_00_00_00_00_37218bf94be1667d143b2b514f473fa8
[2022-06-13 12:35:25,122] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2149, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2114, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1544, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1499, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1489, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 137, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Syntax error: Expected end of input but got identifier "CREATE_BQ_TBL_QUERY_fhvhv" at [1:1]

Location: US
Job ID: airflow_gcs_2_bq_dag_bq_create_fhvhv_tripdata_partitioned_table_task_2022_06_12T00_00_00_00_00_37218bf94be1667d143b2b514f473fa8

[2022-06-13 12:35:25,204] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=gcs_2_bq_dag, task_id=bq_create_fhvhv_tripdata_partitioned_table_task, execution_date=20220612T000000, start_date=20220613T123524, end_date=20220613T123525
[2022-06-13 12:35:25,227] {standard_task_runner.py:97} ERROR - Failed to execute job 655 for task bq_create_fhvhv_tripdata_partitioned_table_task (400 Syntax error: Expected end of input but got identifier "CREATE_BQ_TBL_QUERY_fhvhv" at [1:1]

Location: US
Job ID: airflow_gcs_2_bq_dag_bq_create_fhvhv_tripdata_partitioned_table_task_2022_06_12T00_00_00_00_00_37218bf94be1667d143b2b514f473fa8
; 14666)
[2022-06-13 12:35:25,264] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-13 12:35:25,346] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
